aten/src/ATen/Context.h

```diff
@@ -128,6 +130,7 @@ class CAFFE2_API Context {
   bool deterministic_cudnn = false;
   bool benchmark_cudnn = false;
   bool enabled_mkldnn = true;
+  bool release_original_weights = false;
```

Comment:
Should we default this to true on mobile?

Reply:
Yeah, we could. Only issue I see with that is that someone trying out mobile build on server for experimentation purposes might want unpacking to work. But I'm okay with making it default if you want.